backends/dpdk/dpdkArch.h

```diff
@@ -67,6 +67,49 @@ class ConvertToDpdkArch : public Transform {
     }
 };
 
+// DPDK target supports lookahead instruction only with header operand.
```

Comment:
Maybe you can use /// or /**, I think this generates documentation differently.

Reply:
done in 0bf49e4d209d1f46741814913bc7f32627c099e2